Gemfile.lock

```diff
@@ -0,0 +1,762 @@
+PATH
```

Comment:
Is this intentional to commit the .lock file ?

Reply:
No, that is a mistake